util/cosmos/iscn/sign.js

```diff
@@ -49,37 +49,35 @@ function getAuthorISCNPayload(payload) {
       contributionType: 'http://schema.org/author',
       ...author,
     });
-  } else {
-    let entity;
+  } else if (cosmosWallet || userId) {
+    const identifier = [];
     let likePrefixAddress = cosmosWallet;
     if (likePrefixAddress) {
       if (!isValidLikeAddress(cosmosWallet)) {
         likePrefixAddress = changeAddressPrefix(cosmosWallet, 'like');
       }
     }
-    if (author && likePrefixAddress) { // from Keplr
-      entity = {
-        '@id': likePrefixAddress,
-        name: author,
-        description: authorDescription,
-      };
-    } else if (userId && displayName) {
-      entity = {
-        '@id': `${EXTERNAL_URL}/${userId}`,
-        name: displayName,
-      };
-    } else if (likePrefixAddress) {
-      entity = {
-        '@id': likePrefixAddress,
-      };
+    if (userId) {
+      identifier.push({ '@type': 'PropertyValue', propertyID: 'Liker ID', value: `${EXTERNAL_URL}/${userId}` });
```

Comment:
Is using `liker.land/${user_id}` better for a Depub context?

Reply:
this is share across all iscn, so not specific to depub
the question would be should like.co or liker.land be the canonical url for liker id, we assume like.co before